server/datastore/mysql/teams.go

```diff
@@ -308,9 +308,13 @@ func (ds *Datastore) TeamFeatures(ctx context.Context, tid uint) (*fleet.Feature
 	if err := sqlx.GetContext(ctx, ds.reader, &raw, sql, tid); err != nil {
```

Comment:
Please remind me why we need a `**json.RawMessage` as argument.

Reply:
looking with fresh eyes, I'm not sure we do. I think I just mirrored the function that's above this one. Will adjust.

Reply:
No need to change if this already work, am ok with creating a separate issue to take a deeper look.